asciidoctorj-core/src/main/java/org/asciidoctor/ast/AbstractNode.java

```diff
@@ -17,6 +24,7 @@
     boolean isRole();
     boolean hasRole(String role);
     String getRole();
+    String role();
```

Comment:
With Dan we decided to avoid using the short (none Java Beans) form. In fact there is an issue to change everywhere. For this reason there is the getRole() method. The same should be for context, document, ... you can read more at https://github.com/asciidoctor/asciidoctorj/issues/240 We are going to gradually but if new methods can follow this convention it will be less work to be done then. Of course it is totally fine you didn't notice this change yet :)